src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -294,7 +297,16 @@ const EntityTableToolbar = ({
   /**
    * Debounced `updateData` function.
    */
-  const debouncedRefresh = debounce((config) => updateData(config), 800);
+  const debouncedRefresh = useCallback(
```

Comment:
I'm not sure about this debouncing here. When I was working on the duplicate requests issues, the main culprit was this debounce here. see https://github.com/RedHatInsights/insights-inventory-frontend/pull/2129/files#r1722269670